app/styles/ember-arcgis-portal-components.scss

```diff
@@ -259,28 +261,19 @@ div.panel-body:has(.item-picker) {
       background-color: $white;
       padding-top: 15px;
       .full-width-btn {
+        margin-bottom: 15px;
         padding-left: 15px;
-        padding-right: 15px;
-        div {
-          display: table-row;
-          display: block;
-        }
       }
       .side-by-side {
-        display: table;
-        width: 100%;
-        div {
-          display: table-row;
-        }
-        a {
-          display: table-cell;
-          width: 50%;
-          white-space: normal;
+        display: flex;
+        padding-left: 15px;
+        margin-bottom: 15px;
+        align-items: baseline;
+        .back-btn {
+          margin-right: 5px
         }
-        button {
-          display: table-cell;
-          width: 100%;
-          white-space: normal;
+        .preview-btn {
+          margin-left: 5px
```

Comment:
This is nitpicky but I think a better way to do this is to remove `btn-block` from the two buttons inside the `.side-by-side`. Then you would want this on `.side-by-side`:

```
display: flex;
padding-left: 15px;
margin-bottom: 15px;
align-items: baseline;
justify-content: space-between;
```
And you would add `flex: 1` to `.back-btn` and `.preview-btn`.